File: 12. Rekursi/E. Konversi Biner/e_konversi_biner.go
package main

import (
	"fmt"
)

func biner(N int) {
	if N == 1 {
		fmt.Print(1)
	} else {
		biner(N / 2)
		fmt.Print(N % 2)
	}
}
func main() {
	var N int
	fmt.Scan(&N)
	biner(N)
}

File: 12. Rekursi/E. Konversi Biner/e_konversi_biner.java
import java.util.Scanner;

public class Solution {
    public static void biner(int N) {
        if (N == 1) {
            System.out.print(1);
        } else {
            biner(N / 2);
            System.out.print(N % 2);
        }
    }

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int N = scan.nextInt();
        scan.close();
        biner(N);
    }
}
